platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/storage/FileSource.java

```diff
@@ -358,16 +340,6 @@ private static boolean isPathWritable(String path) {
     return new File(path).canWrite();
   }
 
-  private static void lockPathLoaders() {
-    internalCachePathLoaderLock.lock();
-    resourcesCachePathLoaderLock.lock();
-  }
-
-  private static void unlockPathLoaders() {
-    resourcesCachePathLoaderLock.unlock();
-    internalCachePathLoaderLock.unlock();
-  }
-
```

Comment:
If you wonder where the locking/unlocking is done, it's done in `getResourcesCachePath` and `getInternalCachePath`, respectively. No need to duplicate the code from these methods.